.travisci/coverage.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+mvn jacoco:report coveralls:report -DrepoToken=${repoToken}
+mvn jacoco:report-integration
+mvn jacoco:merge -Djacoco.destFile=target/jacoco-aggregate.exe
+mvn jacoco:report -Djacoco.dataFile=target/jacoco-aggregate.exe
+
+(java -jar $HOME/.script/codacy-coverage-reporter.jar report -l Java -r target/site/jacoco/jacoco.xml)
+$HOME/.script/codecov.bash -F"$TRAVIS_OS_NAME$TRAVIS_JDK_VERSION"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/powerunit/powerunit-extensions-matchers/pullRequest?prid=5574175)